docker-compose-cli.yaml

```diff
@@ -5,8 +5,6 @@ services:
     build:
       context: .
       dockerfile: ./packages/app/Dockerfile
-    environment:
-      - VITE_APP_ENVIRONMENT=local
```

Comment:
Not needed anymore. We need to setup `appEnvironment` with runtime and not with build config, since we will use the same built version for different app environments.